db/models/car.capacity.entity.ts

```diff
@@ -0,0 +1,15 @@
+import { AutoIncrement, Column, HasMany, Model, PrimaryKey, Table } from "sequelize-typescript";
```

Comment:
@er-duytn 
Cái entity nó phải nằm trong module mới đúng, logic em chia theo module mà sao entity lại bỏ ra ngoài
https://docs.nestjs.com/techniques/database đọc Next, let's look at the UsersModule:
Với lại mọi code đều phải bỏ trong src, chứ bỏ ở ngoài như ri nó ko dùng được tính năng npm run format và npm lint lint